README.md

```diff
@@ -67,6 +67,17 @@ Questions or need help with setup? Feel free to open an issue here [https://gith
 
 See [CONTRIBUTING](CONTRIBUTING.md).
 
+### Deployment and workflow
+
+* The staging branch `18f-pages-staging` is **automatically deployed** to our staging site: https://pages-staging.18f.gov/designstandards/.
+* The production branch `18f-pages` is **automatically deployed** to our production site: https://pages.18f.gov/designstandards/.
```

Comment:
For our production site, we currently have two: https://pages.18f.gov/designstandards/ and https://playbook.cio.gov/designstandards/. Should we state both? 

18F pages (https://pages.18f.gov/designstandards/) is the faster, more reliable one, but https://playbook.cio.gov/designstandards/ is what is publically known.